clarity/contracts/arkadiko-auction-engine-v1-1.clar

```diff
@@ -439,9 +447,21 @@
       )
       (try! (contract-call? vault-manager redeem-auction-collateral ft token-string reserve (get collateral-amount last-bid) tx-sender))
     )
+    (if (< (get xusd last-bid) (var-get lot-size))
+      (begin
+        (map-set auctions
+          { id: auction-id }
+          (merge auction {
+            lots-sold: (+ u1 (get lots-sold auction)),
```

Comment:
Q: lots-sold is already updated on 'register-bid', why again when trying to redeem?

Reply:
In `register-bid` it is only updated if you bid over lot size (1000 USDA).. in here it is updated when you redeem (unless it was already set as sold since we check that bid < 1000 USDA)